packages/grub-config/build.yaml

```diff
@@ -0,0 +1,8 @@
+requires:
+- name: "base"
+  category: "distro"
+  version: ">=0"
+
+steps:
+- mkdir /etc/cos
+- cp config/grub.cfg config/bootargs.cfg /etc/cos
```

Comment:
do we need to also set the permissions/owner/group here?

Reply:
Good question, I do not have any strong opinion. I don't think it hurts having it readable and this is part of the immutable region, thus there aren't many chances to modify it...